src/host/exe/CConsoleHandoff.cpp

```diff
@@ -0,0 +1,63 @@
+#include "precomp.h"
+
+#include "CConsoleHandoff.h"
+
+#include "srvinit.h"
+
+// Routine Description:
+// - Helper to duplicate a handle to ourselves so we can keep holding onto it
+//   after the caller frees the original one.
+// Arguments:
+// - in - Handle to duplicate
+// - out - Where to place the duplicated value
+// Return Value:
+// - S_OK or Win32 error from `::DuplicateHandle`
+HRESULT _duplicateHandle(const HANDLE in, HANDLE& out)

```

Comment:
`static`. we don't need to leak this symbol outside this file :)
```suggestion
static HRESULT _duplicateHandle(const HANDLE in, HANDLE& out)
```